### DISCUSSION TITLE
Storymap (tiles of Stories as an overview)

### DISCUSSION THREAD

**Discussion by olierxleben** [olierxleben/lore-or-loot#11] [Ideas]
I would like to bring up a discussion about an Overview of stories. 

On the "map" a player gets an overview of available stories. Each story equals one storyfile (.mdx) and related storyfiles can be visualized by providing an author with the possibility to create tiles for their very own stories (a tile needs to be defined on its own)

Process: A PR merged into main including new stories will create new tiles from the meta data of a story.

**Comment by keppla** [comment]
The example i did is in multiple mdx files (due to me having no idea how to structure it ay other way). Is that fine (because you say "equals one storyfile")
